README.md

```diff
@@ -289,16 +279,93 @@ system reboots.
 You'll be able to stop and start Valkey using the script named
 `/etc/init.d/valkey_<portnumber>`, for instance `/etc/init.d/valkey_6379`.
 
-Code contributions
------------------
+# Building using `CMake`
+
+In addition to the traditional `Makefile` build, Valkey supports an alternative, **experimental** build system using `CMake`
+
+To build and install `Valkey`, in `Release` mode (an optimized build), type this into your terminal:
+
+For Linux:
+
+```bash
+mkdir build-release
+cd $_
+cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/opt/valkey
+sudo make -j$(nproc) install
+# Valkey is now installed under /opt/valkey
+```
+
+For macOS:
+
+```bash
+mkdir build-release
+cd $_
+cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/opt/valkey
+sudo make -j$(sysctl -n hw.ncpu) install
+# Valkey is now installed under /opt/valkey
+```
+
+Other options supported by Valkey's `CMake` build system:
+
+## Special build flags
+
+- `-DWITH_TLS=<on|off|module>` enable TLS build for Valkey
+- `-DWITH_RDMA=<off|module>` enable RDMA module build (only module mode supported)
+- `-DWITH_MALLOC=<libc|jemalloc|tcmalloc|tcmalloc_minimal>` choose the allocator to use. Default on Linux: `jemalloc`, for other OS: `libc`
+- `-DWITH_SANITIZER=<address|thread|undefined>` build with address sanitizer enabled
+- `-DBUILD_UNIT_TESTS=[1|0]`  when enabled, the build will produce the executable `valkey-unit-tests`
```

Comment:
BUILD_UNIT_TESTS seems unused, I wasn't able to get the Unit tests to get built.

Reply:
Fixed